packages/openapi-generator/src/parser/schema.ts

```diff
@@ -169,9 +169,17 @@ function parseEnumSchema(
   const type = schema.type ? getType(schema.type) : 'string';
   return {
     type,
-    enum: (schema.enum || []).map(entry =>
-      type === 'string' ? getEnumStringValue(entry) : entry
-    )
+    enum: (schema.enum || []).map(entry => {
+      if (type === 'string' && entry !== null) {
+        return getEnumStringValue(String(entry));
+      }
+      if (entry === null && !schema.nullable) {
+        throw new Error(
+          "'Null' was used as a parameter, but nullable wasn't declared"
```

Comment:
Should we be this restrictive? Shouldn't we rather log a warning, but still add null as a possible value in the enum? @FrankEssenberger @jjtang1985 what are your opinions?

Reply:
I would propose the following:
- If skipValidation is set to `false` throw an execption
- If skipeValidation is set to `true` log a warning and except the not perfect spec files.